seed.js

```diff
@@ -57,65 +58,294 @@ const sizesList = [
   {id: 4, size: 'L'},
   {id: 5, size: 'XL'},
   {id: 6, size: 'XXL'},
-]
+];
+
+const shoeSizesList = [
+  {id: 1, size: '6'},
+  {id: 2, size: '6.5'},
+  {id: 3, size: '7'},
+  {id: 4, size: '7.5'},
+  {id: 5, size: '8'},
+  {id: 6, size: '8.5'},
+  {id: 7, size: '9'},
+  {id: 8, size: '9.5'},
+  {id: 9, size: '10'},
+  {id: 10, size: '10.5'},
+  {id: 11, size: '11'},
+  {id: 12, size: '11.5'},
+  {id: 13, size: '12'},
+];
 
 const userTypesList = [
   {id: 1, userTypes: 'admin'},
   {id: 2, userTypes: 'pending'},
   {id: 3, userTypes: 'regular'},
   {id: 4, userTypes: 'guest'},
   {id: 5, userTypes: 'oAuth'},
-]
+];
+
+const usersList = [
+  {
+    firstName: 'Elham',
+    lastName: 'Amini',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'elhamfarvid@gmail.com',
+    password: 'abcde',
+    userTypeId: 1,
+  },
+  {
+    firstName: 'Johnson',
+    lastName: 'Lin',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'johnsonlin1993@gmail.com',
+    password: 'abcde',
+    userTypeId: 1,
+  },
+  {
+    firstName: 'Svetlana',
+    lastName: 'Rovinsky',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'svetlana.rovinsky@gmail.com',
+    password: 'abcde',
+    userTypeId: 1,
+  },
+  {
+    firstName: 'Jake',
+    lastName: 'Hertz',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'hertzjake@gmail.com',
+    password: 'abcde',
+    userTypeId: 1,
+  },
+  {
+    firstName: 'Mark',
+    lastName: 'Bae',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'mark@fullstackacademy.com',
+    password: 'abcde',
+    userTypeId: 2,
+  },
+  {
+    firstName: 'Eliot',
+    lastName: 'Szwajkowski',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'eliot@fullstackacdemy.com',
+    password: 'abcde',
+    userTypeId: 3,
+  },
+  {
+    firstName: 'Russell',
+    lastName: 'Moore',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'russell@fullstackacdemy.com',
+    password: 'abcde',
+    userTypeId: 5,
+  },
+  {
+    firstName: 'Ryan',
+    lastName: 'Jasinski',
+    billingAddress: '5 Hanover Square, New York, NY 10004',
+    shippingAddress: '5 Hanover Square, New York, NY 10004',
+    email: 'ryan@fullstackacdemy.com',
+    password: 'abcde',
+    userTypeId: 2,
+  },
+  {
```

Comment:
What is line 160 ~ 164 for?

Reply:
It was me testing if we could create a “user” with no info to simulate a guest coming to the site and putting items in their cart